src/iris/bin/sender.py

```diff
@@ -389,7 +389,10 @@ def deactivate():
                 connection.commit()
                 break
         except Exception:
-            logger.exception('Failed running deactivate query. (Try %s/%s)', i + 1, max_retries)
+            if i == max_retries - 1:
```

Comment:
instead of all `i + 1` and `i -1` logic, what if we did `xrange(1, max_retries + 1)` instead of needing to alter `i` every time we reference it?